frontend/src/components/occupationcard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export default function OccupationCard({ occupation }) {
+  if (!occupation) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div className="occupation-card">
+      <h2>{occupation.name} (Code: {occupation.stdCode})</h2>
```

Comment:
let's remove the occupation stdCode from here, the user doesn't need to see that